app/src/main/java/edu/nextstep/camp/calculator/MainActivity.kt

```diff
@@ -1,46 +1,68 @@
 package edu.nextstep.camp.calculator
 
 import android.os.Bundle
+import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import edu.nextstep.camp.calculator.databinding.ActivityMainBinding
+import edu.nextstep.camp.calculator.domain.Calculator
+
 class MainActivity : AppCompatActivity() {
     private lateinit var binding: ActivityMainBinding
 
+    private var expressionTool = Expression()
+    private var calculator = Calculator()
```

Comment:
```suggestion
    private val expressionTool = Expression()
    private val calculator = Calculator()
```
변수를 정의하고나서 해당 변수에 값을 저장하지 않는 경우라면, val을 활용해보면 어떨까요?